src/app/f2f/controllers/root.js

```diff
@@ -58,13 +54,19 @@ class RootController extends BaseController {
   }
 
   async getAddressInfo(axios, req) {
-    const headers = {
-      "x-govuk-signin-session-id": req.session.tokenId,
-    };
-    const res = await axios.get(`${API.PATHS.PERSON_INFO}`, {
-      headers,
-    });
-    return res.data;
+    const tokenId = req.session.tokenId;
+
+    if (tokenId) {
+      const headers = {
+        "x-govuk-signin-session-id": tokenId,
+      };
+      const res = await axios.get(`${API.PATHS.PERSON_INFO}`, {
+        headers,
+      });
+      return res.data;
+    } else {
+      logger.error("Missing sessionID");
```

Comment:
Should this throw an error so it hits the catch i9n the calling method and returns an error